### PR TITLE
17b hackathon post

### PR DESCRIPTION
**Pull request by StefanThoma**

### CODE REVIEW

**Comment by StefanThoma** [comment]
@bms63 Decided to write a review of the hackathon anyways...

**Comment by StefanThoma** [comment]
This `Link` issue should be resolved once pull-request 17b is through (as the date-change will impact the url)

File: posts/2023-06-19_hackathon_writeup/index.qmd [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+```{r}
+#| include: false
+introR$in_which_country_do_you_reside_in %>% unique()
+introR$what_kind_of_organization_are_you_working_for
+```
+
+## *Introduction to R* workshop
+
+We were really excited to see over **500** people from around 40 countries joining our *introduction to R workshop* in January!
```
Comment (kaz462) [inline posts/2023-06-19_hackathon_writeup/index.qmd:59]:
```suggestion
We were really excited to see over **500** people from around 40 countries joining our *Introduction to R workshop* in January!
```

File: posts/2023-06-19_hackathon_writeup/index.qmd [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  mapping = aes(y = what_kind_of_organization_are_you_working_for, fill = what_kind_of_organization_are_you_working_for, x = n)
+) +
+  geom_col() +
+  ylab("") +
+  theme_classic() +
+  theme(legend.position = "none")
+```
+
+
+216 out of `r sum((introR$is_your_company_using_r_for_clinical_trial_data_analysis %>% table(useNA = "always"))[1:2])` confirmed that their company is already using R for clinial trial data analysis, the remaining 131 did not answer this question.
```
Comment (kaz462) [inline posts/2023-06-19_hackathon_writeup/index.qmd:78]:
```suggestion
216 out of `r sum((introR$is_your_company_using_r_for_clinical_trial_data_analysis %>% table(useNA = "always"))[1:2])` confirmed that their company is already using R for clinical trial data analysis, the remaining 131 did not answer this question.
```

File: posts/2023-06-19_hackathon_writeup/index.qmd [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+The target audience for this workshop were programmers who are very familiar with SAS, but not so familiar with R, our polls confirmed this.
```
Comment (kaz462) [inline posts/2023-06-19_hackathon_writeup/index.qmd:80]:
```suggestion
The target audience for this workshop was programmers who are very familiar with SAS, but not so familiar with R, our polls confirmed this.
```

File: inst/WORDLIST.txt [hunk 4]
```diff
@@ -138,12 +138,79 @@ WIP
...
 ADTTE
 ADVS
+aes
+bb
+ben
+bindCache
+bindEvent
+challange
+checkin
+clinial
```
Comment (kaz462) [inline inst/WORDLIST.txt:157]:
```suggestion
```

**Review by kaz462 — CHANGES_REQUESTED** [review]

**Comment by StefanThoma** [comment]
@cicdguy do you know why the CICD Links workflow fails? 
There are no errors due to failing links, I don't know where the error arises. 
Thanks!

**Review by kaz462 — APPROVED** [review]

**Pull request merged**